### PR TITLE
fix(rockets): Throw if trying to create JPEG without libjpeg-turbo

### PR DESCRIPTION
**Pull request by karjonas**

### CODE REVIEW

File: plugins/Rockets/ImageGenerator.cpp [hunk 1]
```diff
@@ -102,6 +102,7 @@ ImageGenerator::ImageJPEG ImageGenerator::createJPEG(
     frameBuffer.unmap();
     return image;
 #else
+    throw std::runtime_error("Need libjpeg-turbo; cannot create any image");
```
Comment (hernando) [inline plugins/Rockets/ImageGenerator.cpp]: No return needed.

Comment (karjonas) [reply plugins/Rockets/ImageGenerator.cpp]: done

**Comment by karjonas** [comment]
retest this please

**Comment by karjonas** [comment]
retest this please

**Pull request merged**